lab-justin/test/lab03.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const hexPractice = require(__dirname + '/../lib/lab03');
+const expect = require('chai').expect;
+
+describe('test module lib/lab03.js', function(){
+  describe('test fx index()', function(){
+    it('should return hm?', function(done){
```

Comment:
Put meaningful statements in your describe and it blocks so that your tests can be understood by other people too.